Content.Server/Flash/FlashSystem.cs

```diff
@@ -224,4 +224,18 @@ public FlashAttemptEvent(EntityUid target, EntityUid? user, EntityUid? used)
             Used = used;
         }
     }
+
+    public sealed class FlashEvent : CancellableEntityEventArgs
+    {
+        public readonly EntityUid Target;
+        public readonly EntityUid? User;
+        public readonly float Duration;
+        public readonly float Strength;
+
+        public FlashEvent(EntityUid target, EntityUid user)
+        {
+            Target = target;
+            User = user;
+        }
```

Comment:
This shouldn't be cancellable as it doesn't do anything in code and tbh it should just be a struct event raised by-ref anyway.